### DISCUSSION TITLE
Text search over CommonPool

### DISCUSSION THREAD

**Discussion by sedol1339** [mlfoundations/datacomp#65] [Announcements]
Is there any web UI for performing text search over CommonPool? I want to use if to collect my own dataset for few-shot vision tasks.

(I know there is already a search engine over LAION-5B, but I need the search not to be model-assisted to avoid model bias. In LAION-5B, this is not the case, because it searchs by CLIP image embeddings, and LAION-5B itself is model-filtered. So searcing over CommonPool seems better solution).

To implement this by myself, I guess I need some search engine. Would you recommend Apache Solr or other? Any common practices here?

**Comment by rom1504** [comment]
Depending on your goals, maybe the best way is to build an index with
clip-retrieval over the provided clip embeddings

**Comment by jvpassarelli** [comment]
Working on laion search website [at this url](https://d1o3cl9ccvfpb2.cloudfront.net/) - hope it is useful - [documenting effort here](https://github.com/jvpassarelli/laion-image-search)
